doc/decisions/frontend_mocking_tool.md

```diff
@@ -0,0 +1,42 @@
+# Frontend API Mocking Tool
+
+## Problem
+
+We need to select an API mocking tool for our frontend development that allows us to simulate API responses for testing and development purposes.
+
+## Constraints
+
+- The selected tool should seamlessly integrate with our frontend development environment.
+
+## Assumptions
+
+## Considered Alternatives
+
+- **MSW (Mock Service Worker):** [MSW](https://mswjs.io/) is a popular API mocking tool for react applications. It intercepts requests on network level so that code can be tested the same way it is executed in production. Frontend devs can also use msw for development so that they don't have to wait for the backend to be ready.
+
+- **Nock:** [Nock](https://github.com/nock/nock) is also a very popular alternative for API mocking and according to some developers in forums, it is easier to set up than MSW.
+   MSW, however, seems to be a more React-centric approach and Nock can only be used with Node.js and not in the browser.
```

Comment:
```suggestion
- **MSW (Mock Service Worker):**
  [MSW](https://mswjs.io/) is a popular API mocking tool for react applications. It intercepts requests on network level so that code can be tested the same way it is executed in production.
  Frontend devs can also use msw for development so that they don't have to wait for the backend to be ready.

- **Nock:**
  [Nock](https://github.com/nock/nock) is also a very popular alternative for API mocking and according to some developers in forums, it is easier to set up than MSW.
  MSW, however, seems to be a more React-centric approach and Nock can only be used with Node.js and not in the browser.
```